mocca-client/src/test/java/com/paypal/mocca/client/MoccaClientQueryTest.java

```diff
@@ -59,9 +59,9 @@ public void queryInvalidRequest() {
             fail("should throw a validation exception");
         } catch (EncodeException e) {
             assertTrue(e.getCause() != null);
-            assertEquals(e.getMessage(), "Constraint violations found in request parameter 'sampleRequest'");
+            assertEquals(e.getMessage(), "Constraint violations found in request parameter:");
```

Comment:
Isn't the name of the parameter missing in the end of the String?

Reply:
hmm. I should probably rewrite this message. The exception now should have **all** of the constraint violations but the messages won't refer to our var names unless we add some special processing to do the mapping. Instead they look like `getOneValidSample.arg0.bar` [method|argument position|bean field name] in this case.